src/test/java/org/opensearch/neuralsearch/search/query/HybridCollectorManagerTests.java

```diff
@@ -1169,4 +1169,41 @@ public void testScrollWithHybridQuery_thenFail() {
             illegalArgumentException.getMessage()
         );
     }
+
+    @SneakyThrows
+    public void testCreateCollectorManager_whenPaginationDepthIsEqualToNullAndFromIsGreaterThanZero_thenFail() {
+        SearchContext searchContext = mock(SearchContext.class);
+        // From >0
+        when(searchContext.from()).thenReturn(5);
+        QueryShardContext mockQueryShardContext = mock(QueryShardContext.class);
+        TextFieldMapper.TextFieldType fieldType = (TextFieldMapper.TextFieldType) createMapperService().fieldType(TEXT_FIELD_NAME);
+        when(mockQueryShardContext.fieldMapper(eq(TEXT_FIELD_NAME))).thenReturn(fieldType);
+        TermQueryBuilder termSubQuery = QueryBuilders.termQuery(TEXT_FIELD_NAME, QUERY1);
+        // if pagination_depth ==0 then internally by default it will pick 10 as the depth
```

Comment:
is this correct statement, I think we deleted te default value of 10 for pagination